docs/source/getting_started/overview.rst

```diff
@@ -48,6 +47,22 @@ JupyterLab is served from the same `server
 `notebook document format <http://nbformat.readthedocs.io/en/latest/>`__ as the
 classic Jupyter Notebook.
 
+.. _stability:
+
+Stability of JupyterLab
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The current beta release of JupyterLab is stable for daily use.  
+Likewise, all future releases in the beta series will be stable for daily use.
+
+.. _classic:
+
+Classic Jupyter Notebook
```

Comment:
IMHO, I'd consider a title that addresses how the question in the FAQ connects to the classic notebook